game/templates/game/board.html

```
@@ -279,12 +286,12 @@
 
             <!-- Rest stays the same -->
             <div id="modeSelection" style="display: flex; flex-direction: column; gap: 14px;">
-                
+
                 <button class="btn btn-gold" id="welcomePvPBtn" style="padding: 12px; font-size: 1.05rem;">Pass & Play
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)